docs/UserGuide.md

```diff
@@ -203,6 +205,9 @@ Done! I just cleared all items for you.
 ____________________________________________________________
 ```
 
+Note:
+> 📄 Clearing all orders will add back items belonging to the orders back to the system. <br>
```

Comment:
Hi, do update this part as mentioned in the previous PR, think you didnt manage to branch properly and the changes in the previous PR is shown here as well